### PR TITLE
Added EL9 instructions

### PR DESCRIPTION
**Pull request by mambelli**

### CODE REVIEW

**Review by vitodb — COMMENTED** [review]
I added comments with some change suggestion for the install process for EL9.
DE configuration templates for EL9 have been added in the contrib repo.
The section "Setup pressure-based pilot submission" assumes that DE is installed via RPMs, as it uses `systemctl` to start/stop decisionengine service.
Maybe we should add a note to mention the different way to start/stop decisionengine in case it is installed via PIP?

**Comment by vitodb** [comment]
`podman` is leaking volumes each time it starts a container, in the long run this is exhausting system resources.
To check current volumes used by podman user can run `podman volume list`.
To clean up volumes user can run `podman volume prune -f` after all podman container have been stop and removed.

In instructions the procedure we have to stop/rm podman container is
```
podman stop decisionengine-redis | xargs podman rm
```
maybe we can replace that with
```
podman stop decisionengine-redis
podman rm decisionengine-redis
```
that could be more user friendly.

File: doc/source/install.rst [hunk 1]
```diff
@@ -53,69 +53,94 @@ The default postgresql installed on RH7 is 9.2 which is outdated. Suggest to rem
...
+
+1. Make sure the correct repositories and priorities are set. ::
+
+    # CRB ("Code Ready Builder" - PowerTools ) is used for swig and other dependencies
+    dnf install -y yum-utils
+    dnf config-manager --set-enabled crb
+    # EPEL is used for OSG dependencies
+    yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+    # OSG is used for GlideinWMS, HTCSS and others
+    yum install https://repo.opensciencegrid.org/osg/3.6/osg-3.6-el9-release-latest.rpm
```
[the remark this answers is not in the document]
Comment (mambelli) [reply doc/source/install.rst:103]: done

File: doc/source/install.rst [hunk 2]
```diff
@@ -53,69 +53,94 @@ The default postgresql installed on RH7 is 9.2 which is outdated. Suggest to rem
...
-    vi /etc/yum.repos.d/osg-upcoming.repo
-    vi /etc/yum.repos.d/osg-contrib.repo
-    # Change the Epel repository priority to make sure that comes after the OSG repositories, which are 98. Make sure that epel has:
-    priority=99
-    vi /etc/yum.repos.d/epel.repo
 
-   The complete version of the GlideinWMS installation instructions is available `here <https://opensciencegrid.org/docs/other/install-gwms-frontend>`_
 
-2. Setup the decision engine yum repositories ::
+Install via RPMs
```
[the remark this answers is not in the document]
Comment (mambelli) [reply doc/source/install.rst:125]: Added note. To be fixed later

File: doc/source/install.rst [hunk 3]
```diff
@@ -155,6 +180,49 @@ Install Decision Engine and the standard modules
...
+
+Skip this if you did the RPM installation. This PIP installation is recommended for development whnen you want to clone the Git repository and change the code.
+There are a few extra steps (dependencies installation ansd setups) that are automated in the RPM installation.
+
+1. GlideinWMS (3.10.x) and HTCondor (aka HTCSS) are needed for Decision Engine. The ``glideinwms`` packages will pull all the other dependencies.
+  The complete version of the GlideinWMS installation instructions is available `here<https://opensciencegrid.org/docs/other/install-gwms-frontend/>`.
+  For a minimal installation, you can use the following command: ::
+
+    dnf install glideinwms-vofrontend-libs glideinwms-vofrontend-glidein glideinwms-userschedd glideinwms-usercollector
+
```
[the remark this answers is not in the document]
Comment (mambelli) [reply doc/source/install.rst:197]: For now I added the command. Will check the GWMS packaging, core should not be needed.

**Pull request merged**